tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature

```diff
@@ -29,34 +29,35 @@ Feature: restrict Sharing
 
   @smokeTest
   Scenario: Restrict users to only share with users in their groups
+#   shares-only-with-groups-members configuration is not available yet in the oCIS webUI
```

Comment:
```suggestion
     # shares-only-with-groups-members configuration is not available yet in the oCIS webUI
```

Reply:
my IDE auto-generate this indentation. Does this auto-generate comment indentation wrong?

Reply:
The comment is for the step below so I would expect that the comment aligns with the step's indentation.